### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by aman11s**
- update category details in the backend file.
- add category context
- add homepage screen.

### CODE REVIEW

File: src/components/Hero.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+export const Hero = () => {
+  return (
+    <>
+      <div className="hero-section">
+        <div className="overlay">
+          <h1>GET FIT</h1>
+          <p>Your one stop fitness solution</p>
+          <a href="#category" className="pt-1">
```
Comment (prajapati-priyanka) [inline src/components/Hero.jsx:10]: Here instead of using `anchor` tag, you should use `Link` tag.

Comment (prajapati-priyanka) [inline src/components/Hero.jsx:10]: Here, instead of using `anchor` tag,  use `Link` tag.

**Review by prajapati-priyanka — APPROVED** [review]
Just check that one suggestion. Rest of the code is very clean. UI looks superb. Cards looks good !

File: src/components/Hero.jsx (hunk 1 above)
Comment (aman11s) [reply src/components/Hero.jsx:10]: Thanks for the suggestion. After I create product listing page I'll change this to `Link` tag.

**Review by SK02K1 — APPROVED** [review]
- Everything is looking good.
- Just some enhancement can be done, like extracting out all the action types into a separate object as a constant and passing object as payload will help you when your project will grow.

Great work @aman11s  🙌

**Comment by aman11s** [comment]
> * Everything is looking good.
> * Just some enhancement can be done, like extracting out all the action types into a separate object as a constant and passing object as payload will help you when your project will grow.
> 
> Great work @aman11s 🙌

**Pull request merged**